tests/snmp/test_snmp_phy_entity.py

```diff
@@ -536,32 +1102,133 @@ def test_thermal_info(duthosts, enum_rand_one_per_hwsku_hostname, snmp_physical_
         position = int(entity_info['position_in_parent'])
         expect_oid = CHASSIS_MGMT_SUB_ID + DEVICE_TYPE_CHASSIS_THERMAL + position * DEVICE_INDEX_MULTIPLE + \
             SENSOR_TYPE_TEMP
-        assert expect_oid in snmp_physical_entity_info, 'Cannot find thermal {} in physical entity mib'.format(
-            name)
+        assert expect_oid in snmp_physical_entity_info, (
+            "Cannot find thermal {} in physical entity mib. "
+            "Redis DB key: {}. Redis DB value: {}. "
+        ).format(
+            name,
+            key,
+            thermal_info
```

Comment:
print expect_oid and snmp_physical_entity_info out. 
Don't print key and thermal_info.